### ISSUE TITLE
[Suggestion] Mouse Injector for GoldenEye 007 and Perfect Dark

### ISSUE THREAD

**Issue by SilentMRG** [pj64team/Project64-Legacy#130]
Hello...

Graslu added Mouse support using the "MouseInjectorDolphinDuck" project files: https://github.com/garungorp/MouseInjectorDolphinDuck
However, he used the 1964 emulator: https://github.com/Graslu/1964GEPD
Which has Jabos plugins dated 2005 and 2011, and they are a nightmare on low-performance hardware.

Looking at Project64 Legacy I saw that the Jabos plugins were updated, at least they are dated 2022. I tested both emulators and saw that P64L worked better with Jabos 1.6.2, which does not happen with Jabos from 1964 Emulator.

In other words, the emulator 1964 Jabos has a drop in frame rate and greatly sacrifices visual quality, while the P64L has a stable frame rate and does not sacrifice visual quality. I'm not kidding, the Jabos version of the P64L even looks like it's running on OpenGL 3.3. It's actually D3D8 very well optimized for potatoes.

Anyway, would it be possible to add the Injector Mouse to the P64L like Graslu did with the 1964? If so, could you do this work for the aforementioned games?

Thank you and best regards!
